src/cljs/org/broadinstitute/firecloud_ui/utils.cljs

```diff
@@ -67,10 +67,10 @@
                     (aset xhr (name k) v))
                   xhr))
           call-on-done (fn []
-                         ((:on-done arg-map) {:xhr xhr
-                                              :status-code (.-status xhr)
-                                              :success? (and (>= (.-status xhr) 200)
-                                                             (< (.-status xhr) 300))}))]
+                         (on-done {:xhr xhr
```

Comment:
This was just a little harmless oversight.  `(:on-done arg-map)` is already bound to `on-done` from the `let`, so no need to get it out of the map again.